contents/blog/dev-marketing-paid-ads.md

```diff
@@ -0,0 +1,154 @@
+
+Awareness means you want people to see your ad. Maybe they won't click on it - that's ok. Your objective is to just get the word out so that someone is exposed to your product enough times that at some point they Google 'super cool product' and come sign up. Awareness-based ads are a small part of _creating_ demand. This takes more effort to measure, but is totally possible - see below.  
+
+We split our budget 50-50 between the two. This is not a 'rule' - just don't spend it all on conversion. 
+
+### Paid ads don't work on developers
+
+Everyone who works at a tech company basically thinks they are immune to ads because they are super good at internetting, and have never seen an ad for a new product, clicked on it, and then signed up. Your _quantitative_ attribution data in PostHog or whatever analytics tool you are using will seem to back this up. 
+
+This is why we ask all users where they heard about PostHog whenever they sign up or book a demo - it's a simple (optional) free text field. Enough of our users say 'ad on Google' or similar that we know paid ads do actually reach a large chunk of them. Especially ones with money to spend on behalf of their company. However, you need to take the time to collect and digest this _qualitative_ attribution data in the first place. 
+
+### An agency will solve all your problems
+
+I'm a big fan of hiring an agency if you're a startup - paying $5-10k/mo for a small, outsourced team is way more efficient at this stage than hiring one paid ads specialist. Look in Europe - US agencies can be 2-3x more expensive. We use [Hey](https://www.heydigital.co/) and like them. 
+
+_However,_ I really recommend you get personally familiar with each of the main channels I've listed below first. Working with an agency isn't just set and forget - you need to constantly give feedback, ask questions, and suggest new ideas if you want to get the most out of them. 
+
+I personally like [the Demand Curve program](https://www.demandcurve.com/growth-program) – it helped get us off the ground and means we're much better equipped to give our agency feedback. Outsource the legwork, not your critical thinking. 
+
+## Channel-by-channel tips
+
+This is not a comprehensive guide on how to set up each channel (use Demand Curve or similar for this) - these are just some non-obvious things we wish we'd known at the start. 
+
+### Google Search
+
+- Good for conversion, bad for awareness.
+- Do not let Google enable any of their bullshit AI-type suggested/dynamic/whatever keyword generator tools - they always result in weirdly worded ads that were clearly not written by a human. Use your _own_ research about what keywords your target user is likely to search for, not what Google suggests. 
+- If you have an SEO article that ranks well (top 3) make sure you aren't bidding on the same keywords - it's wasted money.
+- Bidding on your own brand may feel like a waste of money, but it's super cheap and defends you against competitor ads. 
```

Comment:
Any advice on bidding against competitors brands?

Reply:
I intentionally didn't want to mention that here as then the section risks becoming a whole guide on how to do Google Search (which is obv massive in itself)